assets/js/SignPanel.tsx

```diff
@@ -164,6 +174,15 @@ class SignPanel extends React.Component<
     });
   }
 
+  requestUngroup(): void {
```

Comment:
What do you think about calling this `confirmUngroup`? My initial expectation was that "request ungroup" is what presents the "confirming ungroup" in the first place.

Reply:
I see what you mean, but `confirmUngroup` sounds to me like the function you call to open the confirmation, not the function that performs the ungroup once confirmed. Maybe `performUngroup` is the way to go.